automation/check-patch.packages

```diff
@@ -5,14 +5,11 @@ dnf-command(builddep)
 git
 grubby
 libffi-devel
-libguestfs-devel
 libvirt-devel
 libguestfs-devel
 libxslt-devel
 openssl-devel
 pandoc
 python2-devel
 python2-dulwich
-python2-virtualenv
-python2-tox
```

Comment:
First, +1.
Second, `tox` is somewhat special because it requires a python interpreter(which obviously can't be installed via pip), I think tox's RPM should actually pull the available interpreters, but need to check that. As we currently only test 2.7, it doesn't matter, but in the future it will probably be better to install it from the RPM.
Could be that similar arguments apply to setuptools.